index.js

```diff
@@ -90,7 +90,7 @@ function generate (str, opts) {
 
   // use `facebook/regenerator` for node < v0.11 support
   // TODO: don't use regenerator if native generators are supported...
-  js = regenerator(js, { includeRuntime: false });
+  js = regenerator.compile(js, { includeRuntime: false }).code;
 
   // filename of the pac file for the vm
   var filename = opts && opts.filename ? opts.filename : 'proxy.pac';
```

Comment:
At that point, we might as well just pluck out the entire `regeneratorRuntime` object, in case it's used for other stuff.